src/libguac/encode-webp.c

```diff
@@ -192,10 +192,10 @@ int guac_webp_write(guac_socket* socket, guac_stream* stream,
         return -1;
 
     /* Add additional tuning */
-    config.lossless = lossless;
-    config.quality = quality;
-    config.thread_level = 1; /* Multi threaded */
-    config.method = 2; /* Compression method (0=fast/larger, 6=slow/smaller) */
+    config.lossless = 1; /* force lossless */
```

Comment:
The `guac_webp_write()` function should definitely not ignore its `lossless` parameter entirely, as that's essentially crippling a library function. If lossless WebP should be used across the board wherever WebP would be used, it should be the surface implementation making that decision, not the internals of the generic WebP writer.